src/vde_backend/retrieveDB.py

```diff
@@ -185,6 +206,12 @@ def processArguments():
 		help="day in YYYY_MM_DD format"
 	)
 
+	argparser.add_argument(
+		"--date_range", type=str,
```

Comment:
It is easier to code and use if you split this in to arguments: `--start` and `--end`.